packages/web-pkg/src/components/ContextActions/ActionMenuItem.vue

```diff
@@ -88,6 +88,7 @@ export default defineComponent({
     const componentProps = computed(() => {
       const properties = {
         appearance: props.appearance,
+        ...(props.action.appearance && { appearance: props.action.appearance }),
```

Comment:
That's what I actually do with the empty trashbin action, w/o this code path that wouldn't be possible.

Reply:
I get that, I was proposing switching `appearance: props.appearance` and `...(props.action.appearance && { appearance: props.action.appearance })` so the component prop (if given) has prio over the action.

Reply:
But the prop is given and would overwrite the filled icon. So your suggestion would break the current empty trashbin button appearance (it would be raw)

Reply:
Change it as well

Reply:
Ahh okay. It's missing in the acceptance criteria in the issue, but then it makes sense I guess.